bundles/AdminBundle/Resources/public/js/pimcore/settings/email/log.js

```diff
@@ -431,5 +466,113 @@ pimcore.settings.email.log = Class.create({
         this.grid.store.reload();
         this.grid.getView().refresh();
 
+    },
+
+    getForwardEmailWindow: function (data) {
+        if (data) {
+            var emailType = data.emailLogExistsHtml ? 'html' : 'text';
+            var emailPreviewUrl = '/admin/email/show-email-log?id=' + data.id + '&type=' + emailType;
+
+            var win =  new Ext.Window({
+                width: 800,
+                height: 600,
+                modal: true,
+                title: t("email_log_forward"),
+                layout: "fit",
+                closeAction: "close",
+                items: [{
+                    xtype: "form",
+                    bodyStyle: "padding:10px;",
+                    itemId: "form",
+                    items: [
+                        {
+                            xtype: 'hiddenfield',
+                            name: 'id',
+                            value: data.id
+                        },
+                        {
```

Comment:
I think we only need one field here, for the email address to forward the email to, all other fields such as subject, from address and reply to should be taken from the original email. "To", CC and Bcc should be clared before sending.

Reply:
I have kept "subject", "from", "replyTo" as display fields for reference. Only "To" is required to forward the email.